graphics/lvgl/lv_conf/Kconfig

```diff
@@ -0,0 +1,1110 @@
+                        bool "enable sbit cache"
+                        default n
+                    config LV_FREETYPE_CACHE_FT_FACES
+                        int "The maximum number of FT_Face(0: use defaults)"
+                        default 0
+                    config LV_FREETYPE_CACHE_FT_SIZES
+                        int "The maximum number of FT_Size(0: use defaults)"
+                        default 0
+                endif
+            endmenu
+        endif
+
+        config LV_USE_RLOTTIE
+            bool "Lottie library"
+
+        config LV_USE_FFMPEG
+            bool "FFmpeg library"
+        config LV_FFMPEG_DUMP_FORMAT
+            bool "Dump format"
+            depends on LV_USE_FFMPEG
+            default n
+    endmenu
+
+    menu "Others"
+        config LV_USE_SNAPSHOT
+            bool "Enable API to take snapshot"
+            default y if !LV_CONF_MINIMAL
+
+        config LV_USE_MONKEY
+            bool "Enable Monkey test"
+            default n
+
+        config LV_USE_GRIDNAV
+            bool "Enable grid navigation"
+            default n
+
+        config LV_USE_FRAGMENT
+            bool "Enable lv_obj fragment"
+            default n
+
+        config LV_USE_IMGFONT
+            bool "draw img in label or span obj"
+            default n
+
+        config LV_USE_MSG
+            bool "Enable a published subscriber based messaging system"
+            default n
+
+        config LV_USE_IME_PINYIN
+            bool "Enable Pinyin input method"
+            default n
+        config LV_IME_PINYIN_USE_K9_MODE
+            bool "Enable Pinyin input method 9 key input mode"
+            depends on LV_USE_IME_PINYIN
+            default n
+        config LV_IME_PINYIN_K9_CAND_TEXT_NUM
+            int "Maximum number of candidate panels for 9-key input mode"
+            depends on LV_IME_PINYIN_USE_K9_MODE
+            default 3
+        config LV_IME_PINYIN_USE_DEFAULT_DICT
+            bool "Use built-in Thesaurus"
+            depends on LV_USE_IME_PINYIN
+            default y
+            help
+                If you do not use the default thesaurus, be sure to use lv_ime_pinyin after setting the thesauruss
+        config LV_IME_PINYIN_CAND_TEXT_NUM
+            int "Maximum number of candidate panels"
+            depends on LV_USE_IME_PINYIN
+            default 6
+            help
+                Set the maximum number of candidate panels that can be displayed.
+                This needs to be adjusted according to the size of the screen.
+    endmenu
+
+    menu "Examples"
+        config LV_BUILD_EXAMPLES
+            bool "Enable the examples to be built"
+            default y if !LV_CONF_MINIMAL
+    endmenu
+
+    menu "Demos"
+        config LV_USE_DEMO_WIDGETS
+            bool "Show some widget"
+            default n
+        config LV_DEMO_WIDGETS_SLIDESHOW
+            bool "Enable slide show"
+            depends on LV_USE_DEMO_WIDGETS
+            default n
```

Comment:
Somehow the slideshow option is not working for the Widgets demo, could you please confirm if this also happens on your side?

Reply:
`LV_DEMO_WIDGETS_SLIDESHOW` is not referenced in the lvgl widgets demo, this option is invalid.

Reply:
It is indeed referenced in several places:
```
./graphics/lvgl/lvgl/lv_conf_template.h:#define LV_DEMO_WIDGETS_SLIDESHOW 0
./graphics/lvgl/lvgl/demos/widgets/lv_demo_widgets.py:LV_DEMO_WIDGETS_SLIDESHOW = 0
./graphics/lvgl/lvgl/demos/widgets/lv_demo_widgets.py:    if LV_DEMO_WIDGETS_SLIDESHOW == 0:
./graphics/lvgl/lvgl/demos/widgets/lv_demo_widgets.py:#    if LV_DEMO_WIDGETS_SLIDESHOW:
./graphics/lvgl/lvgl/demos/README.md:#define LV_DEMO_WIDGETS_SLIDESHOW  0
./graphics/lvgl/lvgl/src/lv_conf_internal.h:#ifndef LV_DEMO_WIDGETS_SLIDESHOW
./graphics/lvgl/lvgl/src/lv_conf_internal.h:    #ifdef CONFIG_LV_DEMO_WIDGETS_SLIDESHOW
./graphics/lvgl/lvgl/src/lv_conf_internal.h:        #define LV_DEMO_WIDGETS_SLIDESHOW CONFIG_LV_DEMO_WIDGETS_SLIDESHOW
./graphics/lvgl/lvgl/src/lv_conf_internal.h:        #define LV_DEMO_WIDGETS_SLIDESHOW 0
./graphics/lvgl/lv_conf/Kconfig:        config LV_DEMO_WIDGETS_SLIDESHOW
./examples/lvgldemo/lv_demos/src/lv_demo_widgets/lv_demo_widgets.c:#if LV_DEMO_WIDGETS_SLIDESHOW
./examples/lvgldemo/lv_demos/src/lv_demo_widgets/lv_demo_widgets.c:#if LV_DEMO_WIDGETS_SLIDESHOW
./examples/lvgldemo/lv_demos/src/lv_demo_widgets/lv_demo_widgets.c:#if LV_DEMO_WIDGETS_SLIDESHOW == 0
./examples/lvgldemo/lv_demos/src/lv_demo_widgets/lv_demo_widgets.c:#if LV_DEMO_WIDGETS_SLIDESHOW
./examples/lvgldemo/lv_demos/src/lv_demo_widgets/lv_demo_widgets.c:#if LV_DEMO_WIDGETS_SLIDESHOW
./examples/lvgldemo/lv_demos/lv_ex_conf_templ.h:#define LV_DEMO_WIDGETS_SLIDESHOW  0
```

Reply:
Yes, in LVGL v7 it is used, but in LVGL v8 there is no adaptation for this
```bash
src/lv_conf_internal.h:2345:#ifndef LV_DEMO_WIDGETS_SLIDESHOW
src/lv_conf_internal.h:2346:    #ifdef CONFIG_LV_DEMO_WIDGETS_SLIDESHOW
src/lv_conf_internal.h:2347:        #define LV_DEMO_WIDGETS_SLIDESHOW CONFIG_LV_DEMO_WIDGETS_SLIDESHOW
src/lv_conf_internal.h:2349:        #define LV_DEMO_WIDGETS_SLIDESHOW 0
lv_conf_template.h:729:#define LV_DEMO_WIDGETS_SLIDESHOW 0
docs/CHANGELOG.md:1353:- `lv_animing` new widget for simple slideshow animations
Kconfig:1064:        config LV_DEMO_WIDGETS_SLIDESHOW
demos/widgets/lv_demo_widgets.py:7:LV_DEMO_WIDGETS_SLIDESHOW = 0
demos/widgets/lv_demo_widgets.py:135:    if LV_DEMO_WIDGETS_SLIDESHOW == 0:
demos/widgets/lv_demo_widgets.py:234:#    if LV_DEMO_WIDGETS_SLIDESHOW:
demos/README.md:17:#define LV_DEMO_WIDGETS_SLIDESHOW  0
```

Reply:
You're right, for some reason there was a leftover files from the `v7.3.0` lv_demos on my tree.